tox.ini

```diff
@@ -1,16 +1,21 @@
 [tox]
-envlist = py{27,34,35,36,37}, lint, docs
+envlist =
+    py35-sa13,
+    py{27,36,37,38,39}-sa{13,14},
+    lint,
+    docs
 
 [testenv]
 passenv = PGPASSWORD
-commands = py.test {posargs}
+commands = py.test -s {posargs}
```

Comment:
```suggestion
commands = pytest {posargs}
```

Reply:
Any reason this is important to change back? `-s` allows `ipdb` to be used with `tox`.

Reply:
You can pass -s with posargs in development, other

This -s c also hanges how the capsys fixture works

Also ops should be set in the pytest config file with addopts